components/organism/nestedCheckboxes/src/index.js

```diff
@@ -6,14 +6,17 @@ import MoleculeCheckboxField from '@s-ui/react-molecule-checkbox-field'
 
 const baseClass = 'sui-OrganismNestedCheckboxes'
 
-const checkItemIsChecked = ({checked}) => checked === true
+const checkItemIsChecked = ({props}) => {
+  const {checked} = props
+  return checked === true
+}
 
 const OrganismNestedCheckboxes = ({
+  children,
```

Comment:
What happens if children is not provided? Shouldn't this be required?
And thus, avoid using all over the optional chaining?

Reply:
It wouldn't crash because I used optional chaining to avoid it, but you are right I will make it required.